packages/polling/src/poll.ts

```diff
@@ -347,7 +326,13 @@ export class Poll<T = any, U = any, V extends string = 'standby'>
   private _state: IPoll.State<T, U, V>;
   private _tick = new PromiseDelegate<this>();
   private _ticked = new Signal<this, IPoll.State<T, U, V>>(this);
-  private _timeout: any = -1;
+  /*
+   * If set, the timeout handle for this poller. We use this complex formulation
+   * for the type since the NodeJS environment has a different return type
+   * (`NodeJS.Timeout`) from the browser environment (`number`) and we want this
+   * to compile in both places.
+   */
+  private _timeout?: ReturnType<typeof setTimeout>;
```

Comment:
nice

Reply:
for the record I [learned this from Stack Overflow](https://stackoverflow.com/a/56239226/39808)